lib/routes/overwatch-cn/namespace.ts

```diff
@@ -0,0 +1,7 @@
+import type { Namespace } from '@/types';
```

Comment:
Use existing namespace https://github.com/DIYgod/RSSHub/blob/1e246f25a422cc132d41108b23a0cc122d050bd2/lib/routes/blizzard/namespace.ts#L4 instead of creating a new one.
The site doesn't have a [second level domain](https://docs.rsshub.app/joinus/new-rss/start-code#creating-namespace) of `overwatch-cn` at all.

Reply:
@TonyRL Thank you Tony. I have updated the code and PR information. I hope you can continue to give me some suggestions.